src/tests/modify.test.js

```diff
@@ -298,3 +306,112 @@ describe('modify() - basic mutations', () => {
     });
   });
 });
+
+describe('modify() - reoder fields', () => {
+  it('reorder fields - basic usage as "start"', () => {
+    const baseExample = {
+      properties: {
+        /* does not matter */
+      },
+      'x-jsf-order': ['field_a', 'field_b', 'field_c', 'field_d'],
+    };
+    const result = modify(baseExample, {
+      order: () => {
+        return {
+          order: ['field_c', 'field_a', 'field_b'],
+          // rest: 'end', default behavior
+        };
+      },
+    });
+
+    // 💡 Note how the missing field (field_d) was added to the end as safety measure.
+    expect(result).toMatchObject({
+      'x-jsf-order': ['field_c', 'field_a', 'field_b', 'field_d'],
+    });
+  });
+
+  it('reorder fields - basic usage as "end"', () => {
+    const baseExample = {
+      properties: {
+        /* does not matter */
+      },
+      'x-jsf-order': ['field_a', 'field_b', 'field_c', 'field_d'],
+    };
+    const result = modify(baseExample, {
+      order: () => {
+        return {
+          order: ['field_c', 'field_a', 'field_b'],
+          rest: 'start',
```

Comment:
A different naming approach that I'm thinking:

```
return {
  fields: ['field_c', 'field_a', 'field_b'],
  rest: 'start',
}
```

I usually associate order with desc/asc, so we maybe this naming would be more explicit?

Reply:
Oh good suggestion, I'll apply it!

Reply:
Done in the latest commit